<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showAccountBalance</class>
 <widget class="QDialog" name="showAccountBalance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>607</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 234, 227);
color: rgb(110,110,110);
text-transform: uppercase;
</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>731</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="TxtLabel_tilitapahtumat">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>561</width>
     <height>231</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>561</width>
     <height>231</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,255);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="nappi_kirjaudu_ulos">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nappi_sulje_ikkuna">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>sulje ikkuna</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>621</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248,248,248);
border-radius: 20px;</string>
   </property>
   <widget class="QLabel" name="label_saldo_otsikko">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 186, 111);
background-color: rgb(248,248,248);
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>SALDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TxtLabel_omistajan_tiedot">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>540</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>540</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248,248,248);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248,248,248);
letter-spacing: 0.5px;
color: rgb(135, 203, 185);</string>
    </property>
    <property name="text">
     <string>Tilin 5 viimeisint√§ tapahtumaa:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>TxtLabel_tilitapahtumat</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>nappi_kirjaudu_ulos</zorder>
  <zorder>nappi_sulje_ikkuna</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
